serverlessworkflow/modules/ROOT/pages/use-cases/orchestration-based-saga-pattern.adoc

```diff
@@ -1 +1,269 @@
-//= Orchestration-based SAGA pattern
\ No newline at end of file
+= Orchestration-based SAGA pattern
+=====================================================
+06-27-2022
+:compat-mode!:
+// Metadata:
+:description: Saga Pattern use case example
+:keywords: kogito, workflow, serverless, saga, saga pattern, consistency, microservices, orchestration, orchestrator
+
+== Brief introduction to Saga pattern
+
+Saga is a design pattern to manage data consistency across participants in distributed transaction scenarios.
+
+In a microservice architecture, a participant could be defined as a microservice, responsible to perform actions related to its business domain.
+Saga is meant to provide transaction management using a sequence of steps.
```

Comment:
I was going to say to add a link to a reliable source explaining the Saga Pattern, but I couldn't find one. :(

Reply:
Oh, that's a nice reference!

Reply:
Yeah, I'll add a reference here to the paper.